Lib/test/test_grammar.py

```diff
@@ -1211,6 +1211,8 @@ def testAssert2(self):
 
         self.check_syntax_warning('assert(x, "msg")',
                                   'assertion is always true')
+        self.check_syntax_warning('assert(False, "msg")',
+                                  'assertion is always true')
```

Comment:
Could you also add to test that this doesn't warn on `assert False, "msg"`.

These tests shouldn't be guarded by  `@unittest.skipUnless(__debug__, ...)`.
The warning should be issued regardless of `__debug__`.
See below.

Reply:
Done.